### PR TITLE
Added Q3, W1L1 (2017)

### PR DESCRIPTION
**Pull request by imasaru**
I need to finish L2-7 later...

### CODE REVIEW

**Comment by VitalikL** [comment]
Thanks @imasaru . This quarter has `14` lessons and not `13`. `create.js` script can accept optional count number to generate appropriate number of weeks

**Comment by imasaru** [comment]
Alrighty. Will do 👍

File: src/ja/2017-03/01/01.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title:  今週のテーマ
+date:   24/06/2017
+---
+
+### 今週のテーマ
```
Comment (VitalikL) [inline src/ja/2017-03/01/01.md]: @imasaru With the release of iOS and design principles, we are omitting putting the title as header within the content of the lesson, check out `en` or any other language for `2017-03`

File: src/ja/2017-03/info.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+  title: "ガラテヤの信徒への手紙における副音"
+  description: "すいませんけど、記述が当分の間オンラインにありません。"
+  human_date: "2017年3期の聖書研究ガイド"
+  start_date: "24/06/2017"
+  end_date: "22/09/2017"
```
Comment (VitalikL) [inline src/ja/2017-03/info.yml]: @imasaru Check out `color_primary` and `color_primary_dark` in the `info.yml` for `en`

**Comment by imasaru** [comment]
I'm going to rebase my fork. I messed it up somehow. I will reopen this PR with the requested changes soon...

**Pull request merged**